src/style.css

```diff
@@ -1 +1,54 @@
 /* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 0;
+}
```

Comment:
To avoid potential margin collapse, be consistent with your margins. Add only top or bottom, but not both. In this case, you have a `margin-bottom: 0;` which might not be necessary.